hack/common.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
+SSP_DIR="$(readlink -f $(dirname $0)/../)"
+WORK_DIR="/go/src/github.com/kubevirt/ssp-operator"
+REGISTRY=${REGISTRY:-quay.io/vparekh}
```

Comment:
I'm not sure about using your personal quay repo as a default. Maybe for now for testing, but then please add a TODO comment here, so we don't forget to change it in a future PR.

Reply:
It was a leftover, changed it to Kubevirt now